packages/mobile/scripts/run_e2e.sh

```diff
@@ -124,28 +128,28 @@ if [ $PLATFORM = "android" ]; then
   startPackager
 
   NUM_DEVICES=`adb devices -l | wc -l`
-  if [ $NUM_DEVICES -gt 2 ]; then 
+  if [ $NUM_DEVICES -gt 2 ]; then
     echo "Emulator already running or device attached. Please shutdown / remove first"
     exit 1
   fi
 
   echo "Starting the emulator"
-  $ANDROID_HOME/emulator/emulator -avd $VD_NAME -no-boot-anim &
+  $ANDROID_HOME/emulator/emulator -avd $VD_NAME -no-boot-anim -netdelay $NET_DELAY &
 
   echo "Waiting for device to connect to Wifi, this is a good proxy the device is ready"
   until [ `adb shell dumpsys wifi | grep "mNetworkInfo" | grep "state: CONNECTED" | wc -l` -gt 0 ]
   do
-    sleep 3 
+    sleep 3
   done
 
   runTest
 
   echo "Closing emulator (if active)"
-  adb devices | grep emulator | cut -f1 | while read line; do adb -s $line emu kill; done
+  #adb devices | grep emulator | cut -f1 | while read line; do adb -s $line emu kill; done
```

Comment:
why comment this out?

Reply:
oops! was not supposed to add this file. I am commenting this out while testing so the emulator stays open after something fails.